corehq/apps/analytics/tasks.py

```diff
@@ -168,6 +171,21 @@ def track_confirmed_account_on_hubspot(webuser):
         })
 
 
+@task(queue="background_queue", acks_late=True, ignore_result=True)
+def track_entered_form_builder_on_hubspot(webuser, cookies, meta):
+    _link_account_with_cookie(HUBSPOT_FORM_BUILDER_FORM_ID, webuser, cookies, meta)
```

Comment:
I think we want to rename this function now that you are using it to do more than just linking accounts and :cookie: 
`_send_form_to_hubspot` or something